src/test-styling.md

```diff
@@ -14,61 +14,75 @@ This page demonstrates all the different styling elements used throughout the si
 ## FAQ-Style Q&A Block
 
 <article class="faq-main-content">
-  <header class="faq-question">
-    <h2>What is this test page for?</h2>
-  </header>
+  <h1>What is this test page for?</h1>
 
-  <div class="faq-answer">
-    <p>This test page demonstrates all the different callout types, blockquotes, and other styling elements used in the CRA FAQ site. It helps us design and refine the visual appearance of different components.</p>
+  <p>This test page demonstrates all the different callout types, blockquotes, and other styling elements used in the CRA FAQ site. It helps us design and refine the visual appearance of different components.</p>
```

Comment:
This is a great idea. Also smart to document it directly in the page itself.

Worth mentioning it in the README, and possibly even linking to it from the resources footer section.